include/libtorrent/netlink.hpp

```diff
@@ -106,6 +106,9 @@ namespace libtorrent {
 			return sizeof(m_sockaddr);
 		}
 
+		// commented the comparison operators for now, until the
+		// same operators are implemented for sockaddr_nl
+		/*
```

Comment:
not sure if it's strictly "POSIX", http://man7.org/linux/man-pages/man7/netlink.7.html